### ISSUE TITLE
Spaces on both sides of the double slash (//) that begins an end-of-line comment

### ISSUE THREAD

**Issue by egor-n** [shyiko/ktlint#198]
The [JetBrains Kotlin Coding Conventions suggest](https://kotlinlang.org/docs/reference/coding-conventions.html#horizontal-whitespace) placing a space after `//`:

![image](https://user-images.githubusercontent.com/378493/39243277-9eafb52a-4895-11e8-97d3-94b567d374c4.png)

The [Android Kotlin Style guide suggests](https://android.github.io/kotlin-guides/style.html#horizontal) placing a space before and after the double slash (`//`) if it begins an end-of-line comment:

![image](https://user-images.githubusercontent.com/378493/39242963-9576e862-4894-11e8-82d0-259bf9c2b7c4.png)

----

Ktlint should flag lack of spaces around `//`.

**Correct:**

```kotlin
// comment
var debugging = false // comment
var debugging = false // comment//word
```

**Incorrect:**

```kotlin
//comment
var debugging = false// comment
var debugging = false //comment
var debugging = false//comment

**Comment by vanniktech** [comment]
It should not only flag it but also autocorrect it.

**Comment by shyiko** [comment]
Added in https://github.com/shyiko/ktlint/commit/4385ea52274cbaa0b732ca90c27b64a168a5d698 (thanks once again, @JelloRanger!)

**Comment by marciogranzotto** [comment]
what about `//region` and `//endregion` comments on Android Studio? They only work if there are no spaces after the slashes
